### PR TITLE
update profiles

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-cmgg/germline/tree/main/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/seqplorer.config
Comment (matthdsm) [inline conf/seqplorer.config:1]: Maxentscan also needs an extra param

Comment (nvnieuwk) [reply conf/seqplorer.config:1]: What param do you mean?

Comment (matthdsm) [reply conf/seqplorer.config:1]: Maxentscan will never run without tje maxentscan dir param for the plugin

Comment (nvnieuwk) [reply conf/seqplorer.config:1]: We decided to disable it for now. We can take another look later if needed

**Review by matthdsm — APPROVED** [review]

**Pull request merged**